ooniprobe/Model/Database/Measurement.m

```diff
@@ -19,12 +19,12 @@ + (SRKResultSet*)notUploadedMeasurements {
 
 + (SRKResultSet*)selectWithReportId:(NSString*)report_id {
     return [[[Measurement query] where:
-             [NSString stringWithFormat:@"%@ AND report_id = %@", UPLOADED_QUERY, report_id]] fetch];
+             [NSString stringWithFormat:@"%@ AND report_id = '%@'", REPORT_QUERY, report_id]] fetch];
```

Comment:
Is there a library we can use to construct queries instead of concatenating strings?

Reply:
Where takes a string, so you need to construct the string somehow. I prefer to use *define* strings like `UPLOADED_QUERY` so I don't have to copy it around